### ISSUE TITLE
Reproducing results on FEVER test

### ISSUE THREAD

**Issue by rn-v** [microsoft/Transformer-XH#4]
Hi,

Thanks for open sourcing the code!
I was trying to reproduce the results on the FEVER task. 
Ran the scripts as directed in the Readme and changed the test set path in `config_fever.json` from `data/fever_dev_graph.json` to `data/fever_test_graph.json`.

The label accuracy I'm getting with the shared pre-trained weights in this setting is `39.73%` .
Could you please highlight if there's something else that needs to be changed to get the reported results?

**Comment by henryzhao5852** [comment]
Hi,

I think we our pytorch_tranformer=1.0.0, make sure you use the same version to load the model. (I think for training new models, it's ok to use other versions).

**Comment by rn-v** [comment]
Retried with `pytorch_transformers==1.0.0`, still getting `39.02%`.

**Comment by lemminkainen94** [comment]
Hi,

I'm the one who wrote the mail earlier today :)

It's worth mentioning that we both get 78% on the dev set, which matches your paper. But we can't reproduce the result for the test set.

**Comment by henryzhao5852** [comment]
OK, for test set, there is no label available, so this 39% does not mean anything (I guess it gives a pseudo label for every example), you need to submit your predictions to fever leaderboard to get the test results.

**Comment by lemminkainen94** [comment]
What label do you mean? Looking at `fever_test_graph.json ` I can see the label "SUPPORTS" in the first line.

**Comment by henryzhao5852** [comment]
These are the pseudo labels (all examples have the same label to make the format same as dev set).

**Comment by lemminkainen94** [comment]
Thank you for your clarification. I think that resolves our issue :)

**Comment by rn-v** [comment]
Aah. That makes sense. Thanks!
